src/Icicle/Sea/FromAvalanche/Prim.hs

```diff
@@ -165,6 +165,8 @@ seaOfPrimArray p
       -> PDAlloc (prefixOfValType (ArrayT t) <> "put_immutable") Nothing
      PrimArraySwap t
       -> PDFun   (prefixOfValType (ArrayT t) <> "swap") Nothing
+     PrimArrayDel t
+      -> PDFun   (prefixOfValType (ArrayT t) <> "delete") Nothing
```

Comment:
should be able to do it with a memcpy, or just the simplest possible way otherwise. I guess it has to copy and be immutable? hm